File: Venda-distribuida-java/src/main/java/br/ufc/venda/connection/ConnectionFactory.java
package br.ufc.venda.connection;

import java.sql.Connection;
import java.sql.DriverManager;

public class ConnectionFactory {

	private static String url = "jdbc:postgresql://localhost/sd_venda";
	private static String user = "postgres";
	private static String password = "postgres";
	
	public static Connection getConnection(){
		try{
			return DriverManager.getConnection(url, user, password);
		}catch (Exception e) {
			System.err.println("Não foi possível realizar a conexão: "+e.getMessage());
		}
		return null;
	}
	
}

File: Venda-distribuida-python/connection/ConnectionFactory.py
#-*- coding:utf-8 -*-

import psycopg2

class ConnectionFactory():

    @staticmethod
    def getConnection():
        try:
            return psycopg2.connect(host='localhost', user='postgres',password='postgres',dbname='sd_venda_replica')
        except Exception as e:
            print('Não foi possível realizar a conexão:'+str(e))
